pkg/k8shandler/status.go

```diff
@@ -197,9 +208,9 @@ func getReplicaSetNames(node elasticsearch.ElasticsearchStatus) []string {
 
 	replicasetNames := []string{}
 
-	for _, nodeStatus := range node.Nodes {
+	/*for _, nodeStatus := range node.Nodes {
```

Comment:
Should this function be implemented or removed?

Reply:
i believe removed, not sure if this is cruft from the status update